src/checkovInstaller.ts

```diff
@@ -12,44 +15,145 @@ const isCheckovInstalledGlobally = async () => {
     }
 };
 
-const installOrUpdateCheckovWithPip3 = async (logger: Logger): Promise<string | null> => {
+const validateCheckovVersion = (checkovVersion: string | undefined) => {
+    // validates that the version is >= the min version (or it can be null)
```

Comment:
It's probably a personal issue - but I think comments in the codebase are redundant and causing bloat...
the code should be clear enough and self explanatory.
 if it's not, we should invest in making it so and not document it with comments

Just my beliefs, feel free to keep them if you desire 🙏

Reply:
I guess I agree on this one. On the other ones, the package manager versioning logic is not self-explanatory, in my opinion, which is why I was more verbose there.

Reply:
With the simplified logic (no autoupdate), the other comments are not needed, so I'll remove them too